src/rime/lever/switcher_settings.cc

```diff
@@ -86,7 +86,7 @@ void SwitcherSettings::GetAvailableSchemasFromDirectory(const path& dir) {
           }
         }
         config.GetString("schema/description", &info.description);
-        info.file_path = file_path.string();
+        info.file_path = file_path.u8string();
```

Comment:
The `file_path` is used by the caller in native `*A` APIs.

Reply:
> The `file_path` is used by the caller in native `*A` APIs.

that might be confusions to be solved: rime api/levers api some time UTF-8 required( or output ), sometime ANSI , Orz